consensus/polybft/consensus_runtime.go

```diff
@@ -1328,6 +1315,38 @@ func (c *consensusRuntime) BuildRoundChangeMessage(
 	return signedMsg
 }
 
+func (c *consensusRuntime) updateProposerSnapshot(snapshot *ProposerSnapshot, blockNumber uint64) error {
+	const saveEveryNIterations = 5
+
+	c.logger.Info("Update proposal snapshot started", "block", blockNumber)
+
+	from := snapshot.Height
+
+	for height := from; height <= blockNumber; height++ {
+		c.config.proposerCalc.Update(snapshot, height, c.config)
```

Comment:
Handle this error?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1018/commits/5b5dee9d1e9ef84ccabad386f0d57ec61aa962fe